src/tools/lifter/bir_inst_liftingLib.sig

```diff
@@ -118,16 +118,20 @@ signature bir_inst_liftingLib = sig
   structure bmil_m0_BigEnd_Main : bir_inst_lifting;
 
 
-  (* M0_mod instance, little endian, process SP *)
+  (* M0 mod instance, little endian, process SP *)
```

Comment:
Just a comment, no need to do anything here. The underscore was intentional. It's not the model M0, but M0_mod.

Reply:
I've fixed it now.